tests/integration/helm_test.go

```diff
@@ -17,19 +17,19 @@ var (
 	hslogger = capnslog.NewPackageLogger("github.com/rook/rook", "helmSmokeTest")
 )
 
-func TestHelmSmokeSuite(t *testing.T) {
-	suite.Run(t, new(HelmSmokeSuite))
+func TestHelmIntegrationSuite(t *testing.T) {
+	suite.Run(t, new(HelmIntegrationTestSuite))
 }
 
-type HelmSmokeSuite struct {
+type HelmIntegrationTestSuite struct {
```

Comment:
since you're in the `integration` package, you don't need to include it in the name. `HelmSuite` seems sufficient.